meta-luneos/recipes-core/packagegroups/packagegroup-luneos-extended.bb

```diff
@@ -23,13 +23,14 @@ RDEPENDS_${PN} = " \
   pulseaudio-server \
   \
   alsa-conf \
+  ofono \
   tar \
   udev-extraconf \
+  voicecall \
   webos-connman-adapter \
+  webos-telephonyd \
   wireless-tools \
   \
-  ${@bb.utils.contains('MACHINE_FEATURES', 'phone', 'packagegroup-webos-telephony', '', d)} \
-  ${@bb.utils.contains('MACHINE_FEATURES', 'cellular', 'packagegroup-webos-telephony', '', d)} \
```

Comment:
That would be an option too, or just completely delete the packagegroup-webos-telephony file altogether.... Just didn't test what would happen if I were to delete it. I guess it should be fine, since there are no references to it anymore.